sdcm/sct_events/continuous_event.py

```diff
@@ -67,6 +67,19 @@ def del_event(self, event: ContinuousEvent):
     def find_continuous_events_by_hash(self, continuous_hash: int) -> list[ContinuousEvent]:
         return self.hashed_continuous_events.get(continuous_hash, []).copy()
 
+    def find_running_disruption_events(self):
+        all_nemeses = [(event[0].begin_timestamp, event[0].continuous_hash_tuple[1], event[0].nemesis_name,
```

Comment:
why first convert the events to tuples, and then convert it to a dict ?
why not do it in one pass ?

Reply:
changed